### PR TITLE
[Metricbeat] Migrate ETCD self to use ReporterV2 interface

### PR DESCRIPTION
**Pull request by sayden**
Refer to #10774 for more info

### CODE REVIEW

**Comment by sayden** [comment]
jenkins, test this please

**Comment by sayden** [comment]
jenkins, test this please https://travis-ci.org/elastic/beats/jobs/495522479#L172

**Comment by sayden** [comment]
jenkins, test this again please because `No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.`

**Comment by sayden** [comment]
Error seems unrelated https://beats-ci.elastic.co/job/elastic+beats+pull-request+multijob-windows/beat=filebeat,label=windows/5191/console

File: metricbeat/module/etcd/self/_meta/data.json [hunk 1]
```diff
@@ -9,8 +9,8 @@
             "id": "8e9e05c52164694d",
             "leaderinfo": {
                 "leader": "8e9e05c52164694d",
-                "starttime": "2017-12-07T07:20:33.241031712Z",
-                "uptime": "9m4.919923089s"
```
Comment (jsoriano) [inline metricbeat/module/etcd/self/_meta/data.json]: For another PR, we could convert this to seconds.

Comment (sayden) [reply metricbeat/module/etcd/self/_meta/data.json]: Can you open an issue to enhance this, please?

**Pull request merged**